index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PortFolio Project</title>
+
+  <!-- Linking the page to the style file A.css -->
+  <link rel="stylesheet" href="styles/style.css">
+  
+</head>
+<body>
+  <header class="toolbar">
+    <section class="logo">
+      MZK
+    </section>
+    <section class="menu">
+      <img src="images/Union.png" alt="">
+    </section>
```

Comment:
- Also, to help improve the semantic structure of your site, please use the `nav` tag to wrap this image instead of the `section` tag. Please fix this 😄